canonical-web-and-design/ubuntu.com
# 20.04 homepage takeover and engage page

## Done

- 20.04 homepage takeover and engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/, http://0.0.0.0:8001/takeovers and http://0.0.0.0:8001/engage/20.04-webinars and http://0.0.0.0:8001/engage/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [brief doc](https://docs.google.com/spreadsheets/d/1TkUyXwYB_Ql_g8vTLjZNRpaeWGPGyRq5tUJnHqDnu3A/edit#gid=877682240) and resolve all comments


## Issue / Card

Fixes #7226
